store/modules/actions/payment.js

```diff
@@ -82,7 +83,7 @@ export async function sendISCNSignature(
       publisher,
       memo,
     } = payload;
-    const { txHash, included } = await remoteSignISCNPayload({
+    const { transactionHash } = await signISCNTx({
```

Comment:
above
`const {
      userId,
      displayName,
      cosmosWallet,
      fingerprint,
      title,
      tags,
      type,
      license,
      publisher,
      memo,
    } = payload;`
    
    need to replace 'cosmosWallet' with 'from'
`const {
      userId,
      displayName,
      from,
      fingerprint,
      title,
      tags,
      type,
      license,
      publisher,
      memo,
    } = payload;`